browser/css/menubar.css

```diff
@@ -63,14 +63,60 @@
 	display: none;
 }
 .main-nav {
-	height: 32px; /* on mouseover menubar items, border emerges */
+	height: 36px; /* on mouseover menubar items, border emerges */
 	width: auto;
 	background: #ffffff;
-	padding: 3px 3px 3px 0;
+	padding: 0;
 	white-space: nowrap;
 	z-index: 12;
 	display: flex;
+	align-items: center;
+}
+
+nav:not(.hasnotebookbar) #main-menu-state:not(:checked) ~ #main-menu {
+	display: flex !important;
+}
+
+nav:not(.hasnotebookbar) #main-menu {
+	width: 100%;
+	height: auto;
+}
+
+nav:not(.hasnotebookbar) #menu-last-mod {
+	opacity: 0.7;
 }
+@media screen and (min-width: 700px) {
+	nav:not(.hasnotebookbar) #menu-last-mod {
+		text-align: right;
+		display: flex;
+		justify-content: end;
+		flex-grow: 1;
+	}
+}
+nav:not(.hasnotebookbar) #menu-last-mod > a, nav:not(.hasnotebookbar) #menu-last-mod > a span {
+	text-decoration: none;
+	color: var(--gray-light-txt--color);
+}
+nav:not(.hasnotebookbar) #menu-last-mod > a:hover, nav:not(.hasnotebookbar) #menu-last-mod > a:hover span {
+	text-decoration: none;
+	background-color: transparent;
+	color: var(--gray-light-txt--color);
+}
+@media screen and (min-width: 1200px) {
+	nav:not(.hasnotebookbar).main-nav #document-titlebar {
+		position: absolute;
+		width: auto;
+		left: 50%;
+		transform: translate(-50%, 0);
+		z-index: 1200;
+	}
+}
+@media screen and (max-width: 1199px) {
+	nav:not(.hasnotebookbar).main-nav #document-titlebar {
+		display: none;
+	}
+}
+
```

Comment:
These media queries, some maybe can be in some of the device specific CSS files we have others probably not specially if we want it to react to window resize but then we should test these first in a real tablet (not the responsive mode that can sometimes be tricky) to see if it affects it or not